src/12_scopes.py

```diff
@@ -6,6 +6,7 @@
 
 def changeX():
```

Comment:
I guess with this code we are creating new x variable inside the scope of changeX() function

But we need to change the value of the x outside of the scope of changeX() function.